[IMP] Sales: Updating 'Create quotations' for 19.4 (#18929)
odoo/documentation: content/applications/sales/sales/sales_quotations/create_quotations.rst

Reviewer: ```suggestion
quantities <../invoicing/invoicing_policy>`.
```
Missing a period here.

Reviewer: ```suggestion
For more information about invoicing, refer to :doc:`Invoice based on delivered or ordered
```
"The" is extra, unless there's something like "documentation" at the end of the sentence.

Reviewer: ```suggestion
.. _sales/create_quotations/create-quotation:
```

Reviewer: ```suggestion
.. _sales/create_quotations/send-and-confirm:
```

Reviewer: ```suggestion
   :doc:`optional_products`
```
This was missing an extra space to indent it properly.

Reviewer: ```suggestion
- :guilabel:`Incoterm`: Select an Incoterm (International Commercial Term) to use as predefined
```
Fixing this spelling mistake while we're here.

Reviewer: ```suggestion
In the :guilabel:`Sales` section of the *Other Info* tab, there are sales specific fields that can
be configured.
```
Fixing up tab formatting while we're here.

Reviewer: ```suggestion
In the *Other Info* tab, there are various quotation-related configurations separated into four
different sections: :guilabel:`Sales`, :guilabel:`Delivery`, :guilabel:`Invoicing`, and
```

Reviewer: ```suggestion
Open the *Optional Products* tab to select related products that can be presented to the customer,
which may result in an increased sale.
```

Reviewer: ```suggestion
catalog items can be found in the *Order Lines* tab.
```

Reviewer: ```suggestion
location amongst the items in the *Order Lines* tab. The section heading appears in bold and a
sub-total for all products in a section is displayed. Then, if desired, :ref:`add a Section
```

Reviewer: ```suggestion
In the :guilabel:`Delivery` section of the *Other Info* tab, there are delivery-specific fields that
can be configured.
```

Reviewer: ```suggestion
In the :guilabel:`Invoicing` section of the *Other Info* tab, there are invoicing specific fields
that can be configured.
```

Reviewer: ```suggestion
In the :guilabel:`Tracking` section of the *Other Info* tab, there are tracking-specific fields that
can be configured.
```

Reviewer: ```suggestion
In the *Notes* tab of the quotation form, enter any specific internal notes about the quotation and
customer, if desired.
```

.. image:: create_quotations/quotations-orders-section.png
   :alt: The Quotations and Orders section on the Odoo Sales app Settings page.

- :guilabel:`Mandatory Product`: Prevents sales without a linked product from being completed.
- :guilabel:`Online Signature`: Requests an online signature by default to validate orders. This may
  be changed on individual orders.
- :guilabel:`Online Payment`: Requests online prepayment from customers to confirm orders. The
  payment may be full or partial. When this option is active, an additional field
  (:guilabel:`Payment X %`) appears. A demo with the current payment provider may be configured by
  clicking the :guilabel:`Configure Demo` button. Other providers may be selected by clicking
  :icon:`fa-arrow-right` :guilabel:`View Other Providers`.
- :guilabel:`Default Quotation Validity`: Determines a set amount (in :guilabel:`days`) that
  quotations remain valid for. This default may be changed per order.

> - :guilabel:`Sale Warnings`: Issues warning messages to database users about orders that include'

Reviewer: ```suggestion
- :guilabel:`Sale Warnings`: Issues warning messages to database users about orders that include
```
Stray apostrophe.

  specific products or customers.
- :guilabel:`PDF Quote builder`: Customizes the look of quotations with header pages, product
  descriptions, footer pages, and more.
- :guilabel:`Display Product Images`: Allows product images to be displayed on PDFs and online
  quotations.
- :guilabel:`Lock Confirmed Sales`: Ensures that orders cannot be edited once confirmed.

> - :guilabel:`Pro-Forma Invoice`: Allows pro-forma invoices to be sent to customers ahead of Regular

Reviewer: Out of curiosity, is the "R" in "Regular" meant to be capitalized here? Just want to be sure before you commit and merge.

Author: Good catch! No, it's not.

> - :guilabel:`Pro-Forma Invoice`: Allows pro-forma invoices to be sent to customers ahead of Regular

Author: ```suggestion
- :guilabel:`Pro-Forma Invoice`: Allows pro-forma invoices to be sent to customers ahead of regular
```

  invoices.

To activate any of these settings, click the checkbox beside the desired options. Then, click

> :guilabel:`Save`. Values set to in the :guilabel:`Online Signature`, :guilabel:`Online Payment`,

Reviewer 2: ```suggestion
:guilabel:`Save`. The values set in the :guilabel:`Online Signature`, :guilabel:`Online Payment`,
```

:guilabel:`Default Quotation Validity`, and :guilabel:`Display Product Images` fields are
company-specific. In a multi-company database, these must be set for each individual company.

.. note::
   The exact options available in this section may vary based on the apps and modules currently
   installed in the database.

> .. _sales_quotations/create_quotations/quotations-dashboard:

Reviewer: ```suggestion
.. _sales/create_quotations/quotations-dashboard:
```
I know this may cause issues with other files linking here, but we may as well fix the anchors while we're here.

Order Lines tab
---------------

The first tab on the quotation form is the :guilabel:`Order Lines` tab. In this tab, products may be

> added to the quotation by the *Description* column or through the *Product Catalog*.

Reviewer: ```suggestion
The first tab on the quotation form is the *Order Lines* tab. In this tab, products may be added to
the quotation by the *Description* column or through the *Product Catalog*.
```
Tabs need to use italics.

> To add a product using the *Description* column, simply type the product's name. If it exists in the

Reviewer 2: ```suggestion
To add a product using the *Description* column, type the product's name. If it exists in the
```

database, it will appear as an autocomplete option and can be clicked to configure as desired and

> added to the order. If the product does not exist in the database, simply type in the desired name

Reviewer 2: ```suggestion
added to the order. If the product does not exist in the database, enter the desired name
```

> relevant information like quantity and price.

Reviewer: ```suggestion
database, it appears as an autocomplete option and can be clicked to configure as desired and added
to the order. If the product does not exist in the database, enter the desired name and relevant
information like quantity and price.
```
Just getting rid of the "will" if we can.

.. image:: create_quotations/description-column.png
   :alt: The Description column in a quotation with both a manually added product and the
         autocomplete open.

.. important::
   Products added to sales quotations by manually entering a name and price are **not** added to the
   database. They only exist on the quotation/sales order where they were typed into the
   *Description* column.

To add a product through the *Product Catalog*, click :guilabel:`Catalog` to reveal a separate page
showcasing every item and potential product variant in an organized catalog display. Items are

> organizable by :guilabel:`Product Category` and :guilabel:`Attributes`.

Reviewer 2: ```suggestion
showcasing every item and potential product variant in an organized catalog display. Items can be organized by :guilabel:`Product Category` and :guilabel:`Attributes`.
```

  :guilabel:`Salesperson` is a member of a sales team, that team is auto-populated in the field.
- :guilabel:`Company`: Select a company from the drop-down menu this quotation should be associated
  with. This field only appears when working in a multi-company environment.
- :guilabel:`Online signature`: Click this checkbox to request an online signature from the customer
  to confirm the order. This field only appears if the *Online Signature* setting has been enabled.

> - :guilabel:`Online payment`: Click this checkbox  and enter a desired percentage in the adjacent

Reviewer: ```suggestion
- :guilabel:`Online payment`: Click this checkbox and enter a desired percentage in the adjacent
```
Extra space.
